plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions.php

```diff
@@ -78,11 +78,27 @@ protected function render( $attributes, $content, $block ) {
 
 		$product_type = $product->get_type();
 
+		$slug = $product_type . '-product-add-to-cart-with-options';
+
 		if ( in_array( $product_type, array( ProductType::SIMPLE, ProductType::EXTERNAL, ProductType::VARIABLE, ProductType::GROUPED ), true ) ) {
+			$template_part_path = Package::get_path() . 'templates/' . BlockTemplateUtils::DIRECTORY_NAMES['TEMPLATE_PARTS'] . '/' . $slug . '.html';
+		} else {
+			/**
+			 * Filter to declare product type's cart block template is supported.
+			 * 
+			 * @since x.x.x
+			 * @param boolean $supports Is the type supported
```

Comment:
I guess this `@param` needs to be updated? :slightly_smiling_face: Same for the `@since` version.

Reply:
What @since should we use?